app/src/main/java/com/wafflestudio/snutt2/views/logged_in/home/TableListViewModel.kt

```diff
@@ -61,6 +61,31 @@ class TableListViewModel @Inject constructor(
     }
 
     suspend fun deleteTable(tableId: String) {
+        if (currentTableRepository.currentTable
+            .filterNotNull()
+            .first().id == tableId
+        ) {
```

Comment:
어차피 suspend 함수 내라서 상관없긴 한데, 여기는
```suggestion
        if (currentTableRepository.currentTable.value?.id == tableId) {
```
로 간단하게 쓰는 게 나을듯
밑에 있는 `checkTableThemeChangeable`에서 좀 쓸데없이(?) first()를 썼는데 지금과 같이 현재 값만 볼 때는 그냥 `.value` 쓰면 됨! StateFlow가 이런 걸 하기 위해 있는 거니까

Reply:
StateFlow가 아니라 그냥 Flow였으면, `.value`같은 건 없고 무조건 suspend 함수 내에서 `.first()`로 가져와야 하는데 StateFlow는 코루틴 내부가 아니어도 막 value의 최신 값을 가져올 수 있지